dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -1437,17 +1444,19 @@ private boolean resolveHang(long tasksCompleted, boolean queueEmpty, int poolSiz
             } else {
                 // there's a hang, but we can't add any more threads...  emit a warning the first time this
                 // happens for a given hang, but otherwise just bail
-                if (hangIntervalCounter == 1) {
+                if (hangMaxThreadsMessageEmitted == false && hangIntervalCounter >= 1) {
```

Comment:
>= 1 could be > 0.  It is a nit, but can deliver without it.

Reply:
Good point about > 0 ... changed.
And thanks for catching the other places where somebody had been monkeying with coreThreads setting. I've removed the coreThreads item in the com.ibm.ws.messaging.runtime path. 
But I'm sort of confused about the other three ... there is already nothing going on in there (all commented out) and I'm not sure how much value is added by extending the existing comments.
Seems like the real thing to do would be to delete those defaultInstances.xml files since there is nothing in them any more, and I suppose to delete the OSGI-INF/wlp directories preceding them as well. Is there some reason to not delete all vestiges of the prior kludge?